### DISCUSSION TITLE
Reddit Saved Post User Setting Formatting

### DISCUSSION THREAD

**Discussion by Aston77** [AAndyProgram/SCrawler#128] [Q&A] [answered]
Trying to download a Reddit profile with saved posts but not having luck.

Just to clarify the required setup to get this working.

- Log into account
- Export cookies from browser -> Import cookies into Reddit config settings
- Enter user name of saved posts (I assume just the name and not the full web path)

Is there anything obvious I appear to be missing?
Does it just create a folder in the Reddit folder for that user name just like any other user name?

FWIW - I tried specifying a specific user path for the saved posts and resulted in the same outcome...nothing happens.

![image](https://github.com/AAndyProgram/SCrawler/assets/78501124/7dab2123-1bca-4b69-b651-90984ab0516e)

**Comment by AAndyProgram** [comment, marked as answer]
The saved post user is the username of your profile. In `https://reddit.com/user/UserName1` your username is `UserName1`.
Your settings looks good.
Open the [form for downloading saved posts](https://github.com/AAndyProgram/SCrawler/wiki#saved-posts) and click download.

>FWIW - I tried specifying a specific user path for the saved posts and resulted in the same outcome...nothing happens.

By default, the saved posts path will be created in the root path for this site. In your case, it's `.....SCrawler_2023.5.12.0_x64\Data\Reddit\!Saved\`.
If you want to set up a specific location, you can enter the path in this textbox (Saved posts path).

**Reply by Aston77** [reply]
That did it. I missed the fact/button that saved posts had to be triggered separately than the other posts.

Thanks for the clarification.

**Reply by AAndyProgram** [reply]
You're welcome
